### ISSUE TITLE
Not working in Calendar

### ISSUE THREAD

**Issue by gabmaquignaz** [zenangst/Gray#136]
Not working for Calendar on my machine (Intel 2020 Touchbar MacBook Pro) on macOS Sonoma 14.3.1.

Note that `defaults write com.apple.iCal NSRequiresAquaSystemAppearance -bool yes ` does not work either.

**Comment by zenangst** [comment]
Hey @gabmaquignaz, what is the return value for the `defaults write` command?
My best guess is that it doesn't have access to write that changes.
Apple has tightened OS security a lot during the recent updates.
Still looking into the issue for the new version of Gray.

**Comment by gabmaquignaz** [comment]
It does ask for authorization ("Terminal would like to access data from other apps.") first and then returns nothing like if the command was executed successfully. But then the appearance of Calendar is not updated.
![Capture d’écran 2024-03-18 à 15 10 14](https://github.com/zenangst/Gray/assets/61877971/5ab8dff5-1c57-411e-a93b-3228f054a6da)

**Comment by zenangst** [comment]
You should also make sure that the app isn't running before you execute the command (you probably already made sure of that). Otherwise you might end up setting a value to a plist and the app would then store the previous value on exit as `UserDefaults` is stored in memory while the application is running.

**Comment by gabmaquignaz** [comment]
Yes, I made sure the application being toggled was not running. Other macOS applications like Photos and Mail only work in Light mode with Dark mode system, but not the other way around. I don't know if it helps.

**Comment by zenangst** [comment]
Yeah, the settings is only one way. Your system needs to be in dark mode because what the setting does it to force the app to run in light mode. Apple removed the way to do it in reverse a few OS releases ago.

**Comment by gabmaquignaz** [comment]
That's right. But indeed still does not work for Caldenar, let me know if I can help.
